kgforge/core/archetypes/store.py

```diff
@@ -294,9 +296,10 @@ def _download_one(
     # CR[U]D.
 
     def update(
-        self, data: Union[Resource, List[Resource]], schema_id: Optional[str]
+        self, data: Union[Resource, List[Resource]], schema_id: Optional[str], debug: bool = False
     ) -> None:
         # Replace None by self._update_many to switch to optimized bulk update.
+        catch_exceptions = False if debug else True
```

Comment:
In other places, when calling run, you've directly used catch_exceptions=not debug. any reason for the change in style? (in this file you did it also for register, but then for update, tag, deprecate and freeze, the style changed)

Reply:
Ah yes, I believe I initially used this style, and then I was suggested to change it all, but seems I missed many cases

Reply:
I just saw the suggestion